test/e2e/nix/build_suite_test.go

```diff
@@ -0,0 +1,104 @@
+package nix_test
+
+import (
+	"github.com/benchkram/bob/bob/bobfile"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/benchkram/bob/bob"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	dir     string
+	version string
+
+	stdout *os.File
+	pr     *os.File
+	pw     *os.File
+
+	b *bob.B
+
+	initialPath string
+)
+
+var _ = BeforeSuite(func() {
+	version = bob.Version
+	bob.Version = "1.0.0"
+
+	initialPath = os.Getenv("PATH")
+
+	//Initialize mock bob files from local directory
+	mocks := []string{
+		"with_use_nix_false",
+		"with_bob_dependencies",
+		"with_task_dependencies",
+	}
+	nameToBobfile := make(map[string]*bobfile.Bobfile)
+	for _, name := range mocks {
+		abs, err := filepath.Abs("./" + name)
+		Expect(err).NotTo(HaveOccurred())
+		bf, err := bobfile.BobfileRead(abs)
+		Expect(err).NotTo(HaveOccurred())
+		nameToBobfile[name] = bf
+	}
+
+	testDir, err := ioutil.TempDir("", "bob-test-version-*")
+	Expect(err).NotTo(HaveOccurred())
+	dir = testDir
+
+	err = os.Chdir(dir)
+	Expect(err).NotTo(HaveOccurred())
+
+	//Save bob files in dir to have them available in tests
+	for name, bf := range nameToBobfile {
+		err = bf.BobfileSave(dir, name+".yaml")
+		Expect(err).NotTo(HaveOccurred())
+	}
+
+	b, err = bob.Bob(bob.WithDir(dir), bob.WithCachingEnabled(false))
+	Expect(err).NotTo(HaveOccurred())
+	Expect(bob.CreatePlayground(dir)).NotTo(HaveOccurred())
```

Comment:
Sure we should create the playground here, this creates some coupling? You created your own test environment why not do the full initialization unrelated to the playground?

Reply:
True, is not needed. I copied from another test as I thought we need some directories from playground to be able to run. 

Will remove